### ISSUE TITLE
[Inputs.json]: Talkback announces each edit field twice in 'Start Time' under 'Inputs'.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#88]
### Target Platforms

Android

### SDK Version

2023.12.12.1 (85454887)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Android 13
Device: S20
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.12.12.1 (85454887)
Tool: Logitech Bluetooth Keyboard

**Repro Steps:**
1. Launch the Adaptive cards app on Android.
2. Turn on Talkback.
3. Navigate to 'Import' control and select it.
4. Navigate to 'Sample cards' and select it.
5. Navigate to available dropdown under 'Sample cards'.
6. Navigate to available dropdown under 'Sample cards' and select Input.
7. Navigate all the controls under dropdown and select 'Inputs'.
8. Navigate to 'Start Time' and activate it.
9. Observe the issue.

**Actual Result:**
Talkback announces each edit field twice in 'Start Time' under 'Inputs'. It announces as "4 4" and "54 54".

**Observations:**
This issue is not repro when user navigate through swipe gesture on the Hour and Minute edit fields.

**Expected Result:**
Talkback should announce each edit field only once in 'Start Time' under 'Inputs'. It should announce as "4" and "54".

**User Experience:**
Screen Reader users face difficulty if Talkback announces each edit field twice in 'Start Time' under 'Inputs'.

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/e4bf5abe-71ff-42e7-8d31-7cef64bdd603

![When Talkback focus lands on the Hour and Minutes edit fields it is announcing information twice as 4 4 or 59 59](https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/105f33be-96ba-42ac-84cd-5521fe0205f7)

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Please don't assign bug to any Dev until UnderReview Tag is added.
